### PR TITLE
Quizz result

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
- Add: quiz questions page

Screenshot 2022-02-20 at 6 11 11 PM

- Add Result Page

Screenshot 2022-02-20 at 6 11 56 PM

### CODE REVIEW

**Comment by harshitpaliwal95** [comment]
- preview link for Questions page - https://deploy-preview-2--quizz-bizz.netlify.app/pages/quiz-page/quiz.html

- preview link for Result page - https://deploy-preview-2--quizz-bizz.netlify.app/pages/result-page/result.html

**Comment by ashifkhn** [comment]
Looks Nice, Good to go.

**Review by ashifkhn — COMMENTED** [review]
Good to go.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: pages/category-page/category.html [hunk 1]
```diff
@@ -28,40 +28,44 @@
...
-            <div class="card-comp-contant">
-              <h2 class="card-h2">Naruto Uzumaki</h2>
-              <h3 class="card-h3">Take this quizz to test yourself!</h3>
-              <h3 class="card-h3">5 Question</h3>
+          <a href="../rules-page/rules.html">
+            <div class="card-component naruto-card">
+              <div class="card-comp-img">
+                <img
+                  class="card-top-img"
+                  src="../../assets/naruto.jpg"
```
Comment (shubhmehra) [inline pages/category-page/category.html:56]: suggestion: according to your folder structure you can do /assets/naruto.jpg

File: pages/quiz-page/quiz.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="quiz.css" />
```
Comment (agrawalshanu708) [inline pages/quiz-page/quiz.html:7]: add meta tags of description, author

**Pull request merged**